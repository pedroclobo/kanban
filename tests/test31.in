t 1 tarefa
m 1 pedro DONE
l
q
