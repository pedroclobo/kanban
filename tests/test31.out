task 1
no such user
1 TO DO #1 tarefa
